packages/katello/katello/restore.rb

```diff
@@ -178,8 +184,11 @@ def restore
       FileUtils.chown(nil, 'postgres', @dir) unless @is_foreman_proxy_content
       Dir.chdir(@dir)
 
-      set_file_security
-      reset_katello
+      metadata = YAML.load_file(File.join @dir, 'metadata.yml')
+      unless @options[:incremental] or metadata[:incremental]
```

Comment:
`or` is fine here, but rubocop will complain about this when its moved to foreman-maintain, if you don't mind changing it to `||`, it will save me the trouble! :)

Reply:
absolutely - meant to change these back (had a `not` in other PR)